src/org/zaproxy/zap/extension/alert/ExtensionAlert.java

```diff
@@ -224,6 +229,17 @@ public void run() {
         }
     }
 
+    private static boolean isInvalid(Alert alert) {

```

Comment:
Negative logic on purpose? Does a `isValid` already exist?

Reply:
Yes, to read better in the above if statement. Can change to `isValid` if preferred.

Reply:
I guess it isn't a huge issue, just surprised me.